indra/databases/pubchem_client.py

```diff
@@ -125,6 +126,38 @@ def get_mesh_id(pubchem_cid: str) -> Union[str, None]:
     return pubchem_mesh_map.get(pubchem_cid)
 
 
+def get_substance_mesh_id(pubmed_id: str) -> List[str]:
+    """Return substance MeSH ID for a given PubMedID.
+
+    Parameters
+    ----------
+    pubmed_id :
+        PubMedID ID whose substance MeSH ID will be returned
+
+    Returns
+    -------
+    :
+        Substance MeSH ID's corresponding to the given PubMed paper or
+        if None present or a failed query, an empty list will be returned.
+
+    """
+    url = '%sdb=%s&id=%s&retmode=text&rettype=XML' % (eutils_url, 'pubmed', pubmed_id)
+    res = requests.get(url)
```

Comment:
you can use the `params` kwarg in `requests.get` to do all of that formatting for you. just give it a dictionary of the keys to values (don't forget to remove the ?) from the URL

Reply:
actually, that call shouldn't be necessary, @samuelbunga there is already a function called `get_full_xml` to get an element tree.

Reply:
Oh I just saw this function `get_full_xml`
Thanks @bgyori  and @cthoyt